plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/SectionInterface.php

```diff
@@ -2,12 +2,12 @@
 
 namespace Automattic\WooCommerce\Admin\Features\ProductBlockEditor\ProductTemplates;
 
-use Automattic\WooCommerce\Admin\BlockTemplates\ContainerInterface;
+use Automattic\WooCommerce\Admin\BlockTemplates\BlockInterface;
 
 /**
  * Interface for block containers.
  */
-interface SectionInterface extends ContainerInterface {
+interface SectionInterface extends BlockInterface {
```

Comment:
You still need the ContainerInterface... looks like we have a `BlockContainerInterface` that would be exactly what we want to extend a single interface. (Note: There isn't currently a technical need, since `BlockInterface` is actually a superset of `ContainerInterface`, but if we make changes in the future we would want to make sure that `SectionInterface` is explicitly a `ContainerInterface`, via `BlockContainerInterface`).